.github/workflows/doc-tests.yml

```diff
@@ -24,6 +24,21 @@ jobs:
             args: -W
     steps:
       - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+          cache: 'pip'
+
+      - name: Cache linkcheck results
+        uses: actions/cache@v4
+        if: matrix.builder == 'linkcheck'
+        with:
+          path: .cache/sphinx-linkcheck
+          key: ${{ runner.os }}-linkcheck-${{ hashFiles('**/*.rst', '**/*.md') }}
+          restore-keys: |
+            ${{ runner.os }}-linkcheck-
```

Comment:
```suggestion
```

Let's remove this part, the GH action is reporting nothing to be cached.

Reply:
oh wow 206 B
```
 Cache Size: ~0 MB (206 B)
/usr/bin/tar -xf /home/runner/work/_temp/c45c0910-fc99-4651-9944-a0ea4613c985/cache.tzst -P -C /home/runner/work/tmt/tmt --use-compress-program unzstd
Cache restored successfully
```

Reply:
yeah, removing.